### ISSUE TITLE
Instant death spells at Tier 1

### ISSUE THREAD

**Issue by itamarcu** [mlenser/kryx-rpg-issues#841]
I'd like to discuss instant death effects at low levels.  This is specifically concerning [Disintegrate](https://www.kryxrpg.com/spells/disintegrate), Dissolve, Ravage, Time Ravage - all similar 1-mana spells that deal single-target burst damage and have a phrase such as:

> If this damage leaves the creature with 0 health, the creature and everything it is wearing and carrying, except magic items, are reduced to a pile of fine gray dust. The creature can be restored to life only by means of a resurrection spell cast with 5 mana or a wish spell.

I think it's problematic to allow these spells available in their current form at 1 mana.  It means that both player characters and monsters with spellcasting have the ability to:
* instantly kill a creature that goes down to 0, preventing any healing - making it a "save or suck" spell in a way, because no healing is going to help, and tier 1 characters probably won't have good enough reaction abilities to avoid this.  it's particularly brutal because it's balanced as a single target spell, meaning that your character can suddenly drop from high health.
* permanently kill the creature, in such a way that normal resurrection is impossible, and only 5 mana spells can do it - will extinguish any hopes of getting the character back, because in low level you'd probably just make a new character.  this can really ruin the plans of some players/DMs.
* completely get rid of evidence - overcoming what I think should be a legitimate problem for level 1-4 adventurers

Does anyone else feel like these effects are too extreme for low levels?

If so, I think the solution is to move that paragraph about complete destruction into a 1 mana augment, or as an additional effect of the existing damage augments.

**Comment by mlenser** [comment]
I agree with what you've written. I'd like to solve this, but without:

- Raising the mana cost. I think direct damage spells should be available at low tiers for these themes.
- Removing the thematics

We could put the turning to dust into an augment, but it feels weird.. Not sure how to accomplish this.

**Comment by itamarcu** [comment]
Ideas:
- Augment to turn completely into dust at 0 health, but cause another status (e.g. can't regain health, max health reduced, staggered) if the creature is still alive.
- Basic spell disfigures the creature beyond recognition, if it reaches 0 hp.
- It only turns a Tiny creature to dust at 1 mana, Small at 2 mana, Medium at 3 mana, etc.   
- Only a partial, half-meter-radius portion of the creature turns into dust, with that size increasing with mana (just like with objects).  This still seems like it would insta-kill, though.
- Roll a die, maybe a random object held or worn by the creature is disintegrated, and with small chance it's the creature itself.

**Comment by DalenWBrauner** [comment]
The disintegration effect could, instead of _costing_ an extra mana point, just be locked behind / "come with" the damage augment. In other words, if you augment the damage of the spell to at least 2 mana, then the rest of the thematics apply. 

You could rationalize this as: at 1 mana, the spell is just a bit too weak to do all that. We could even add a "lesser" thematic effect at 1 Mana, just to preserve the flavor.

**Comment by itamarcu** [comment]
Using my first two suggestions it would be:
> ### Disintegrate
> ...
> If this damage leaves the creature with 0 health, the creature is disfigured, losing hair and gaining radiation scars across part of its body.
> ...
> #### Augment
> ***Complete Disintegration***. You can expend 1 additional mana or psi to make the creature staggered if it fails the saving throw.  If the damage leaves the creature with 0 health, the creature and everything it is wearing and carrying, except magic items, are reduced to a pile of fine gray dust. The creature can be restored to life only by means of a *resurrection* spell cast with 5 mana or a *wish* spell.

Then do the same with "Total Dissolution", "Utter Ruin", "Decay Into Oblivion", or similar augment names.

**Comment by mlenser** [comment]
Disintegrate, Dissolve, Ravage, and Time Ravage now only melt creatures at 3 or more mana.
